readme.md

```diff
@@ -253,6 +254,11 @@ Made with Electron.
 - [Architecture overview](https://github.com/ilyavorobiev/atom-docs/blob/master/atom-shell/Architecture.md)
 
 
+## Books
+
+- [Cross Platform Desktop Applications](https://www.manning.com/books/cross-platform-desktop-applications)
```

Comment:
Hi, the book's site has old copy which needs updating. The book's scope was changed early on in the year to cover Electron as well. I'll ask Manning to update the copy on that page.

I'm currently finishing updates to the remaining chapters, the book goes for final review at the end of this month, and is scheduled for deliver at the end of September: 

https://www.amazon.co.uk/Cross-Platform-Desktop-Applications-Paul-Jensen/dp/1617292842/ref=sr_1_1?s=books&ie=UTF8&qid=1465817504&sr=1-1